test/goose/brokers/redis/api_test.clj

```diff
@@ -206,7 +206,7 @@
 (deftest batch-test
   (testing "[redis] batch API"
```

Comment:
Suggestion: add another test, after calling the batch-expiration method, this api will return invalid id as batch data is not present anymore

Reply:
>this api will return invalid id as batch data is not present anymore

To clarify, the api returns `nil` if batch isn't present/called with invalid id.

I've added a test case for this.